elastic.go

```diff
@@ -266,10 +317,15 @@ func restoreDocuments(sb s3Backend, c elastic.Client, keyPath, indexName string,
 			log.Info("End of blob reached")
 			break
 		}
-		for i := 0; i < batchsize; i++ {
+		i := 0
+		for {
 			key := fmt.Sprintf("%v._source", i)
```

Comment:
Note; for these (where there is a standard formatting conversion specifier) I prefer using it, i.e. `%d` in this case to allow the compiler/static analysis to catch it.